linera-execution/src/evm/revm.rs

```diff
@@ -723,17 +737,30 @@ impl<Runtime: ContractRuntime> CallInterceptorContract<Runtime> {
 
 struct CallInterceptorService<Runtime> {
     db: DatabaseRuntime<Runtime>,
+    contract_address: Address,
```

Comment:
Is this address of a contract _being called_ or _callee_? Rename or add a comment